eng/versioning/external_dependencies.txt

```diff
@@ -34,23 +34,23 @@ com.zaxxer:HikariCP;3.4.5
 commons-codec:commons-codec;1.14
 commons-net:commons-net;3.6
 io.fabric8:kubernetes-client;4.10.3
-io.micrometer:micrometer-core;1.5.4
-io.micrometer:micrometer-registry-azure-monitor;1.5.4
-io.micrometer:micrometer-registry-graphite;1.5.4
-io.netty:netty-buffer;4.1.51.Final
-io.netty:netty-common;4.1.51.Final
-io.netty:netty-codec-http;4.1.51.Final
-io.netty:netty-codec-http2;4.1.51.Final
-io.netty:netty-transport;4.1.51.Final
-io.netty:netty-handler;4.1.51.Final
-io.netty:netty-handler-proxy;4.1.51.Final
+io.micrometer:micrometer-core;1.5.5
+io.micrometer:micrometer-registry-azure-monitor;1.5.5
+io.micrometer:micrometer-registry-graphite;1.5.5
+io.netty:netty-buffer;4.1.52.Final
+io.netty:netty-common;4.1.52.Final
+io.netty:netty-codec-http;4.1.52.Final
+io.netty:netty-codec-http2;4.1.52.Final
+io.netty:netty-transport;4.1.52.Final
+io.netty:netty-handler;4.1.52.Final
+io.netty:netty-handler-proxy;4.1.52.Final
 io.netty:netty-tcnative;2.0.31.Final
```

Comment:
Should this version change to `2.0.34.Final`?

Reply:
good catch @alzimmermsft

Reply:
@alzimmermsft , 
Thank you for your check.
Currently, `netty-tcnative` is not managed by `spring-boot-dependencies`.
I created a PR: https://github.com/spring-projects/spring-boot/pull/23502

And this time, I'll update `netty-tcnative`'s version to `2.0.34.Final` manually.